### PR TITLE
Added a menu item

### PR DESCRIPTION
**Pull request by dokun1**
Added a Caesar Salad menu item into the mix

### CODE REVIEW

**Comment by orgcottontee** [comment]
Please add more to your menu!

File: GitMeFood/ContentView.swift [hunk 1]
```diff
@@ -8,14 +8,15 @@
...
 struct ContentView: View {
-    
     var menuItems: [String] = [
         "Chicken Pesto Pizza",
         "Sushi",
         "BBQ Sandwich",
         "Chicken and Rice",
-        "Whipped Brie"
+        "Whipped Brie",
+        "Caesar Salad"
```
Comment (orgcottontee) [inline GitMeFood/ContentView.swift]: This is a great addition

Comment (dokun1) [reply GitMeFood/ContentView.swift]: Thank you! I just wish they didn't have anchovies in them.

**Comment by orgcottontee** [comment]
LGTM!

**Pull request merged**